.azure-pipelines/all.yml

```diff
@@ -21,18 +21,18 @@ resources:
   - container: dd_agent
     image: gcr.io/datadoghq/agent:latest
     ports:
-      - 8127:8126
+    - 8127:8126
```

Comment:
What are these changes from? Can you remove these changes since they aren't relevant to the integration tile?

Reply:
We didn't change this. Must have come from one of the ddev build commands,
I'll put it back